ui/StatusQ/src/StatusQ/Components/StatusCursorDelegate.qml

```diff
@@ -0,0 +1,20 @@
+import QtQuick 2.15
+
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
+    id: root
+
+    color: Theme.palette.primaryColor1
+    implicitWidth: 2
+    implicitHeight: 22
+    radius: 1
+    visible: parent.cursorVisible
```

Comment:
Assumptions about parents is bad practice. I know, the component is meant to be used in the context of `Text` only but still..

```suggestion
```

Reply:
I can introduce a `property bool cursorVisible`, wdyt?

Reply:
Fixed

Reply:
> I can introduce a `property bool cursorVisible`, wdyt?

Why not simply use built-in `visible` property?

Reply:
Because that doesn't work with `SequentialAnimation on visible`

Reply:
Ah, tricky.. would it make sense to rename `cursorVisible` to `blinking` to improve readability?